reactor/http_action.go

```diff
@@ -73,15 +74,23 @@ func (hta *HTTPAction) request(message *ReactorDelivery) error {
 	if !ok {
 		method = "get"
 	}
-	//params := query["params"].(map[string]interface{})
+
+	params := hta.actionInfo.Params["query"].(map[string]interface{})["params"]
+
+	data := url.Values{}
+	for qkey, qval := range params.(map[string]interface{}) {
+		data.Add(qkey, qval.(string))
```

Comment:
Here might be a bug if qval is `nil` (because of the `interface{}`). Maybe that is better:

```go
if qkey == "" {
    continue
} else if qval != nil {
    data.Add(qkey, qval.(string))
} else {
    data.Add(qkey, "")
}
```

Otherwise you might get panic crash on casting `qval`'s `nil` as `string`, or a rubbish if someone will add zero key etc.

Reply:
Done